.github/workflows/run-a-single-node-from-branch.yml

```diff
@@ -0,0 +1,156 @@
+name: 'Run a node with selected configuration. NodeName=\"DevNode-${{ github.actor }}-${{ github.ref }}\"'
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: "Select a network on which You want to run a node"
+        default: "mainnet"
+        required: true
+        type: choice
+        options:
+          - mainnet
+          - gnosis
+          - sepolia
+          - goerli
+          - chiado
+      cl_client:
+        description: "Select Consensus Layer Client to run node against"
+        default: ""
+        required: true
+        type: choice
+        options:
+          - lighthouse
+          - lodestar
+          #- nimbus
+          - prysm
+          - teku
+      cl_custom_image:
+        description: "In case of need to run non-default cl image (different than actually supported by Sedge) put it in there"
+        default: ""
+        required: false        
+      config:
+        description: "Select a config file which will be selected for tests."
+        default: "default.json"
+        required: true        
+        type: choice
+        options:
+          - default.json
+          - fastSync.json
+          - fullSync.json
+          - fuzzer.json
+      timeout:
+        description: "Timeout in hours before triggering the deletion of smoke test instances"
+        default: "24"
```

Comment:
Do I get this right, that by default a vm created with it will have a lifetime of 24 hours? If yes, then it's awesome.

Reply:
@Scooletz Yes - You can extend that. Will also add info that maximum time of node to be running is 72h.